editor/resources/templates/template.script

```diff
@@ -16,6 +16,12 @@ function update(self, dt)
 	-- Remove this function if not needed
 end
 
+function fixed_update(self, dt)
+	-- Add update code here
+	-- Learn more: https://defold.com/manuals/script/
+	-- Remove this function if not needed
+end
+
```

Comment:
The new function (currently only supported in game object scripts)

Reply:
This is probably ok right?

Reply:
At least not on the render script